components/graph/graph.js

```diff
@@ -127,10 +146,7 @@ GraphViewModel.prototype.loadNodesFromApi = function(callback) {
 
       self.edges(edges);
       self.nodes(nodes);
-
-      if (nodes.length > 0) {
```

Comment:
Why did you remove the `if (nodes.length > 0) {` check? Calling `cy()` on `undefined` will throw.